bucket/hatch.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "1.7.0",
+    "description": "Hatch is a modern, extensible Python project manager.",
+    "homepage": "https://github.com/pypa/hatch",
+    "license": "MIT",
+    "depends": "python",
+    "url": "https://files.pythonhosted.org/packages/py3/h/hatch/hatch-1.7.0-py3-none-any.whl",
+    "hash": "efc84112fd02ca85b7bab54f5e2ef71393a98dc849eac9aca390504031f8a1a8",
+    "installer": {
+        "script": [
+            "Push-Location \"$dir\"",
+            "python -m venv hatch",
+            ".\\hatch\\Scripts\\activate",
+            "try {",
+            "    python -m pip install \"$fname\"",
+            "}",
+            "finally {",
+            "    Remove-Item \"$fname\"",
+            "    deactivate",
+            "    Pop-Location",
+            "}"
+        ]
+    },
+    "bin": "hatch\\Scripts\\hatch.exe",
+    "checkver": {
+        "github": "https://github.com/pypa/hatch"
```

Comment:
Try adding `"regex": "/releases/tag/hatch-(?:v|V)?([\\d\\.]+)"` - I had a similar problem when adding Sonic Visualiser. It's to do with the normal GitHub `checkver` not taking into account that projects sometimes prepend their name to release versions.

Reply:
```suggestion
        "github": "https://github.com/pypa/hatch",
        "regex": "/releases/tag/hatch-(?:v|V)?([\\d\\.]+)"
```